costar_hyper/LICENCE.md

```diff
@@ -1,5 +1,4 @@
-Copyright 2018, Andrew Hundt, Ding Yu, Tomi Nikolla. All rights reserved.
-
+Copyright 2017-2018, Chris Paxton, Andrew Hundt, Yotam Barnoy, Ding Yu, The Johns Hopkins University. All rights reserved.
```

Comment:
hey, this change doesn't look right. I wonder what happened

Reply:
I updated the license to match the license under costar_plan because it seemed outdated.